### ISSUE TITLE
Not all buttons are Buttons and they don't indicate disability apart from color in Navigation screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#810]
**Describe the bug**
Buttons don't indicate disability outside color in Navigation screen. They are displayed on screen but are not functional.

**To Reproduce**
1. Go to Navigation screen and select any site.
2. There are different buttons such as "Delete Site", "Copy Site", "Move Site" etc.
3. You will notice that they are actually not buttons and they don't indicate disability outside color.

**Expected behavior**
Disability should be indicated and it should be buttons.
